src/index.css

```diff
@@ -0,0 +1,28 @@
+
+.Todo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #4E8CAB;
+  padding: 20px 0;
+}
+
+.completed {
+  text-decoration: line-through;
+  color: #204051;
+}
+
+.edit-btn, .delete-btn {
+  padding: 10px;
+  font-size: 16px;
+  background-color: #007EA7;
+  color: white;
+  border: none;
+  border-radius: 50%;
+  cursor: pointer;
+  margin-left: 10px;
+}
+
+.edit-btn:hover, .delete-btn:hover {
+  background-color: #FFFF00;
+}
```

Comment:
if they are already in module css you can remove those, 
if not move them to module css